guides/concepts/multi-sig.md

```diff
@@ -26,16 +26,18 @@ Each account can set its own threshold values. By default all thresholds levels
 
 Low Security:
  * [Allow Trust](./list-of-operations.md#allow-trust) operation
- * Used to allow people to hold credit from this account without exposing the key that enables sending payments from this account.
+   * Used to allow people to hold credit from this account without exposing the key that enables sending payments from this account.
+ * [Transaction processing](./transactions.md)
+    * Charging a fee or updating the sequence number for the source account
 
 Medium Security:
  * All other operations
 
 High Security:
  * [Set Options](./list-of-operations.md#set-options) to change the signers or the thresholds
- * Allows you to create a set of signers that give or revoke access to the account.
-
+    * Allows you to create a set of signers that give or revoke access to the account.
 
+For most cases, it is recommended to set thresholds such that `low <= medium <= high`.
```

Comment:
@MonsieurNicolas  do transactions under the `medium` threshold also need to pass the `low` threshold?
If so then we should make that explicit here.